src/main/java/org/spongepowered/common/mixin/core/entity/EntityMixin.java

```diff
@@ -743,7 +730,7 @@ public ItemEntity entityDropItem(final ItemStack stack, final float offsetY) {
             } else {
                 final ItemEntity entityitem = new ItemEntity(this.world, this.posX, this.posY + (double) offsetY, this.posZ, stack);
                 entityitem.setDefaultPickupDelay();
-                this.world.addEntity0(entityitem);
```

Comment:
Add a `TODO` for me to review this bit of code, it might be refactored into oblivion later when entity capturing is reviewed.